tools/convert-for-mobile-desktop/convert.js

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+
+var fs = require('fs');
+var svg2img = require('svg2img');
+var path = require('path');
+const yargs = require("yargs");
```

Comment:
I think this project's js files try to use single quotes if you want to stay colloquial